docs/mechanical/Fabrication.md

```diff
@@ -0,0 +1,3 @@
+# Fabrication Engineer
```

Comment:
Rename file.
File name must depict its contents.
fabrication_Engineer.md for describing fabrication engineers' role.